### PR TITLE
Product Gallery > Prevent error when "layout" is missing in block attributes

### LINKED ISSUE #10783: Product Gallery > Next/Previous buttons block: Error in php file when visiting a Variable product page
## Describe the bug

When visiting a Variable product page, an error appears related to the Next/Previous buttons block.

## To reproduce

1. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Two," "Twenty-twenty Three," etc.
2. On the left-hand side menu, click on Appearance > Editor > Templates
3. Find and select the 'Single Product' template from the list.
4. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
5. Inside the Page editor, click on the '+' button, usually found at the top left of the editing space or within the content area itself, to add a new block.
6. In the block library that pops up, search for the 'Product Gallery' block. Click on it to add the block to the template.
7. Check that the Previous/Next buttons block is displayed on the Editor.
8. Click on Save;
9. Visit a variable product and check that the error is displayed.

## Expected behavior

No error should appear when visiting the Variable product.

## Screenshots

CleanShot 2023-08-30 at 17 11 39@2x

### PR DESCRIPTION
**Pull request by thealexandrelara**
## What
An error is currently occurring when adding the Product Gallery to the page and visiting a variable product page.

Fixes #10783 

## Why
This PR fixes the bug and make the block to work correctly with variable products.

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Two," "Twenty-twenty Three," etc.
2. On the left-hand side menu, click on Appearance > Editor > Templates
3. Find and select the 'Single Product' template from the list.
4. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
5. Inside the Page editor, click on the '+' button, usually found at the top left of the editing space or within the content area itself, to add a new block.
6. In the block library that pops up, search for the 'Product Gallery' block. Click on it to add the block to the template.
7. Check that the Previous/Next buttons block is displayed on the Editor.
8. Click on Save;
9. Visit a variable product and make sure no error is displayed.

* [ ] Do not include in the Testing Notes 
* [ ] Should be tested by the development team exclusively 

## Screenshots or screencast

| Before | After |
| ------ | ----- |
| CleanShot 2023-08-30 at 17 11 39@2x | CleanShot 2023-08-30 at 17 34 34@2x |

## WooCommerce Visibility

Required:

* [ ] WooCommerce Core
* [ ] Feature plugin
* [x] Experimental
* [ ] N/A

### CODE REVIEW

File: src/BlockTypes/ProductGalleryLargeImageNextPrevious.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ protected function render( $attributes, $content, $block ) {
 			$this->get_class_suffix( $context )
 		);
 
-		$alignment_class = $attributes['layout']['verticalAlignment'] ? 'is-vertically-aligned-' . $attributes['layout']['verticalAlignment'] : '';
+		$alignment_class = isset( $attributes['layout'] ) && $attributes['layout']['verticalAlignment'] ? 'is-vertically-aligned-' . $attributes['layout']['verticalAlignment'] : '';
```
Comment (danieldudzic) [inline src/BlockTypes/ProductGalleryLargeImageNextPrevious.php]: I'd simplify:

```suggestion
		$alignment_class = isset( $attributes['layout']['verticalAlignment'] ) ? 'is-vertically-aligned-' . $attributes['layout']['verticalAlignment'] : '';
```

**Review by danieldudzic — CHANGES_REQUESTED** [review]
Good catch!

Noticed the same issue when testing https://github.com/woocommerce/woocommerce-blocks/pull/10782

Added one minor suggestion :)

File: src/BlockTypes/ProductGalleryLargeImageNextPrevious.php (hunk 1 above)
Comment (thealexandrelara) [reply src/BlockTypes/ProductGalleryLargeImageNextPrevious.php]: Done! Thank you for the suggestion!

**Review by danieldudzic — APPROVED** [review]
Works great! 🙌

**Pull request merged**